dev_guide/builds.adoc

```diff
@@ -906,47 +905,60 @@ source:
 
 === Binary Source
 
-Streaming content in binary format from a local file system to the builder is called a `*binary type build*`.
-The corresponding value of `*BuildConfig.spec.source.type*` is `*Binary*` for such builds.
+Streaming content in binary format from a local file system to the builder is
+called a `*binary type build*`. The corresponding value of
+`*BuildConfig.spec.source.type*` is `*Binary*` for such builds.
 
-This source type is unique in that it is leveraged solely based on your use of the `oc start-build`.
+This source type is unique in that it is leveraged solely based on your use of
+the `oc start-build`.
 
 [NOTE]
 ====
-Since binary type builds require content to be streamed from the local file system, it is not possible
-to automatically trigger a binary type build (e.g. via an image change trigger) because the binary files could
-not be provided. Similarly, you cannot launch binary type builds from the web console.
+Binary type builds require content to be streamed from the local file system, so
+automatically triggering a binary type build (e.g. via an image change trigger)
+is not possible, because the binary files cannot be provided. Similarly, you
+cannot launch binary type builds from the web console.
 ====
 
 To utilize binary builds, invoke `oc start-build` with one of these options:
 
-1) `--from-file`:  The contents of the file you specify will be sent as a binary stream to the builder.  Then
-the builder will store the data in a file with the same name at the top of the build context.
+* `--from-file`: The contents of the file you specify are sent as a binary
+stream to the builder. Then the builder stores the data in a file with the
+same name at the top of the build context.
 
-2) `--from-dir` and `--from-repo`:  The contents will be archived and sent as a binary stream to the builder.
-Then the builder will extract the contents of the archive within the build context directory.
+* `--from-dir` and `--from-repo`: The contents are archived and sent as a binary
+stream to the builder. Then the builder extracts the contents of the archive
+within the build context directory.
 
```

Comment:
Above, the two instances of "Then the builder ..." sound weird to my ear.  Suggest: "The builder then ..."